docs(migration): mention "vi.importActual" (#2927)
vitest-dev/vitest: docs/guide/migration.md

Unlike Jest, mocked modules in `<root>/__mocks__` are not loaded unless `vi.mock()` is called. If you need them to be mocked in every test, like in Jest, you can mock them inside [`setupFiles`](/config/#setupfiles).

**Importing the original of a mocked package**

> If you are only partially mocking a package, you might have previously used Jest's function `requireActual`. In vitest, you should replace these calls by calls to the `vi.importActual` function.

Reviewer: ```suggestion
If you are only partially mocking a package, you might have previously used Jest's function `requireActual`. In Vitest, you should replace these calls with `vi.importActual`.
```

Author: Thank you, I adjusted the wording.

```diff
- const { cloneDeep } = jest.requireActual('lodash/cloneDeep')

> + const { cloneDeep } = vi.importActual('lodash/cloneDeep')

Reviewer: `vi.importActual` is asynchronous, so this code will actually fail.

```suggestion
+ const { cloneDeep } = await vi.importActual('lodash/cloneDeep')
```
